modules/ximgproc/src/sparse_match_interpolators.cpp

```diff
@@ -201,13 +207,13 @@ void EdgeAwareInterpolatorImpl::interpolate(InputArray from_image, InputArray fr
     if(use_post_proc)
         fastGlobalSmootherFilter(src,dst,dst,fgs_lambda,fgs_sigma);
 
+    costMap = Mat();
```

Comment:
`costMap.release();` should do the same.

Reply:
done

Reply:
where?

Reply:
Oh sorry. Got the wrong line of code in line 1306 was the same issue

Reply:
There are two extra cases:
 ` = Mat();` => `.release();`